src/modules/vtol_att_control/vtol_att_control_main.h

```diff
@@ -109,24 +110,24 @@ class VtolAttitudeControl
 
 	struct vehicle_attitude_s 				*get_att() {return &_v_att;}
 	struct vehicle_attitude_setpoint_s		*get_att_sp() {return &_v_att_sp;}
-	struct mc_virtual_attitude_setpoint_s 	*get_mc_virtual_att_sp() {return &_mc_virtual_att_sp;}
```

Comment:
This is not the correct fix. The correct fix is to inherit the struct for the topic.